torch/_inductor/lowering.py

```diff
@@ -5339,6 +5342,35 @@ def mul(a, b):
         return make_pointwise(fn)(a, b)
 
 
+def get_constant_value(x: ir.IRNode) -> Optional[ir.Constant]:
+    """Try convert an arbitrary IR node into an ir.Constant value"""
+    if isinstance(x, ir.MutableBox):
+        return get_constant_value(x.data)
+    if isinstance(x, ir.BaseView):
+        return get_constant_value(x.unwrap_view())
+    if isinstance(x, ir.Constant):
+        return x
+
+    loader = x.make_loader()
+    if hasattr(x, "inner_fn_args"):
+        inner_fn_args = x.inner_fn_args()
+    else:
+        size = x.get_size()
+        idx = [sympy_index_symbol(f"i{n}") for n, s in enumerate(size)]
```

Comment:
```suggestion
        idx = [sympy_index_symbol(f"i{n}") for n in range(len(size))]
```

Reply:
Even more, I see this pattern in quite a few places. Do we want to expose it in a coherent API all throughout the IR?

Reply:
Yeah it would be useful if all IR nodes had `inner_fn_args`.